indico/modules/events/registration/controllers/api/misc.py

```diff
@@ -5,10 +5,20 @@
 # modify it under the terms of the MIT License; see the
 # LICENSE file for more details.
 
+from operator import itemgetter
+
 from indico.modules.events.controllers.base import RHProtectedEventBase
+from indico.util.iterables import group_list
 
 
 class RHAPIRegistrationForms(RHProtectedEventBase):
     def _process(self):
         from indico.modules.events.registration.schemas import RegistrationFormPrincipalSchema
         return RegistrationFormPrincipalSchema(many=True).jsonify(self.event.registration_forms)
+
+
+class RHAPIEventSessionBlocks(RHProtectedEventBase):
+    def _process(self):
+        schema = [{'id': sb.id, 'start_dt': sb.start_dt.astimezone(self.event.tzinfo).date().isoformat(),
+                   'fullTitle': sb.full_title} for s in self.event.sessions for sb in s.blocks]
```

Comment:
Mix of `snake_case` and `camelCase` in the same dictionary?

Reply:
At the beginning it was all snake_case, but I changed the full_title to camelCase to be extracted in a javascript map, should I use all snake_case and rename the variable in the map or all camelCase? It's not clear to me tbh

Reply:
One option could be to keep it all snake_case and use `camelizeKeys` to convert it to camelCase when you fetch the data

Reply:
Or just use snake_case throughout, this makes it easier to search for the key name

Reply:
Yeah, but can't use snake_case in the map function to extract the value, linter error, I think I needed `camelizeKeys`, will try